client/components/data/query-products-list/index.jsx

```diff
@@ -11,7 +11,7 @@ import { isProductsListFetching as isFetching } from 'state/products-list/select
 import { requestProductsList } from 'state/products-list/actions';
 
 class QueryProductsList extends Component {
-	componentWillMount() {
+	componentDidMount() {
```

Comment:
Now that I think about it, no idea. It made more sense to me at that point, but it's all the same.